src/testdir/test_vim9_import.vim

```diff
@@ -3694,4 +3694,42 @@ def Test_import_member_initializer()
   v9.CheckScriptSuccess(lines)
 enddef
 
+def Test_import_name_conflict_with_local_variable()
+  var lines =<< trim END
+    vim9script
+
+    export class Foo
+      def Method()
+        echo 'method'
```

Comment:
Can you change this method to return a number or a string and remove the echo?

Reply:
OK.  I updated the patch.